plugins/action_generate_review_assignment_records/src/generateReviewAssignments.ts

```diff
@@ -459,17 +459,22 @@ const getNewOrExistingAssignmentStatus = (
   canMakeFinalDecision: boolean,
   isSelfAssignable: boolean,
   sectionCodes: string[],
-  existingAssignment?: ExistingReviewAssignment,
+  existingAssignment?: ExistingReviewAssignment
 ): AssignmentState => {
   const isReviewAssigned = existingReviewsAssigned.length > 0
   const isAssigned = existingReviewsAssigned.some(
     ({ userId }) => userId === existingAssignment?.userId
   )
-  // temporarily final decision shouldn't be locked if there are other reviewAssignemt assigned
+  // temporarily final decision shouldn't be locked if there are other reviewAssignment assigned
```

Comment:
Fixed one tiny typo in here, the rest is all auto-formatting (not sure why it was wrong previously)